src/python/turicreate/_deps/README.md

```diff
@@ -0,0 +1,134 @@
+# Lazy import layout
+
+---
+
+## raitionale
```

Comment:
Please run spell check on this whole document. For instance, `raitionale` should be `rationale`.

Reply:
my bad. I rely on google too much and realize my vscode doesn't have google spell check.